build/check.js

```diff
@@ -34,10 +35,15 @@ import conf from './conf';
 gulp.task('check', ['lint', 'build', 'test', 'integration-test:prod']);
 
 /**
- * Checks the code quality of Dashboard. In addition a local kubernetes cluster is spawned.
- * NOTE: This is meant as an entry point for CI jobs.
+ * Waits until local kubernetes cluster is up and running and then build Dashboard.
```

Comment:
Hmm... Previous documentation note was correct. I dont see a reason to change it to this one. Can you revert? And only change implementation.